docs/source/torch.compiler_profiling_torch_compile.rst

```diff
@@ -4,9 +4,9 @@ Profiling to understand torch.compile performance
 What to use torch.profiler for:
 -------------------------------
 
-torch.profiler is helpful for understanding the performance of your program at a kernel-level granularity - for example, it can show graph breaks and GPU utilization at the level of the program. The data provided by the profiler can often help users understand where to investigate further to understand model performance.
+torch.profiler is helpful for understanding the performance of your program at a kernel-level granularity - for example, it can show graph breaks and GPU, CPU and XPU utilization at the level of the program. The data provided by the profiler can often help users understand where to investigate further to understand model performance.
```

Comment:
```suggestion
torch.profiler is helpful for understanding the performance of your program at a kernel-level granularity - for example, it can show graph breaks and GPU, CPU or XPU utilization at the level of the program. The data provided by the profiler can often help users understand where to investigate further to understand model performance.
```